.travis.yml

```diff
@@ -117,3 +117,17 @@ jobs:
         - make edgeimage ARCH="arm64v8"
         - make edgesiteimage ARCH="arm64v8"
         - make bluetoothdevice_image
+
+  deploy:
```

Comment:
This is debatable, my initial thought is this should be a tool which help to build deb or rpm packages, not necessary to be defined in travis.
We'd come up some ideas what's the benefit shall give us if this is defined as a stage in the travis.

Reply:
I think he want to create a deploy stage! This should be executed when we want to build a new deployment. (please correct me if I'm wrong()

Reply:
@chendave Deploy stage of the travis would compile and create the Debian package and upload it to GitHub releases. So, that whenever there's a major version release 1.2, 1.3 a new Debian packages are automatically created.